pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
Both Dima and I came into this change set asking for the same things: the ability to separate the ITs and be able to easily run one (minicluster only) or the other (standalone capable). I would imagine that Mike D. has similar interests as he's talked to me about running the standalone tests. As such, I believe this *is* representative, but I'm not about to start an argument with you over what you consider strongly enough justified.